lib/esi.rb

```diff
@@ -70,12 +73,12 @@ module Esi
     timeout: 60,
     client_id: nil,
     client_secret: nil,
-    cache: nil,
+    cache: ActiveSupport::Cache::MemoryStore.new,
```

Comment:
I don't know how I feel about this... this makes caching opt-out instead of opt-in

Reply:
perhaps default to `:null_store` instead?

Reply:
Also lets use `ActiveSupport::Cache.lookup_store` instead.